### ISSUE TITLE
Allow access to a11y dbus calls

### ISSUE THREAD

**Issue by ndarilek** [flathub/org.mozilla.Thunderbird#8]
As an Orca user, I can't get any speech from Thunderbird. I see the following when running with `--log-session-bus`:

```
C12: -> org.a11y.Bus call org.freedesktop.DBus.Properties.Get at /org/a11y/bus
B13: <- (no sender) return from C12
```

There appear to be some other failing calls as well that might need addressing, but a11y is most crucial to me.

Thanks!

**Comment by TingPing** [comment]
A11y isn't something applications should be granting access to and I believe should work out of the box. What is your distro, flatpak version, etc?

**Comment by ndarilek** [comment]
Fedora Atomic Workstation 27, Flatpak 0.10.4. FWIW, I'm using
non-Flatpak Thunderbird to type this message. The unofficial Firefox
flatpaks work fine too, so maybe you can compare with those?

**Comment by ndarilek** [comment]
It does in fact require permission to talk to the a11y bus. This may be because most apps implement accessibility via GTK modules, but Gecko apps implement it in-process.

The unofficial Firefox flatpak, which works, allows access to the entire system bus. I haven't thoroughly tested this, but granting access to org.a11y.* at least makes this speak via Orca.

PR submitted. Please let me know if there are any additional concerns.
